inyoka_theme_default/templates/portal/usercp/subscriptions.html

```diff
@@ -0,0 +1,110 @@
+{#
```

Comment:
argh, sry, i reviewed the cherry-picked changes, so ignore the following comments, please :)